### PR TITLE
Sidebar: cleanup and enhancements

### PR DESCRIPTION
**Pull request by MaggieCabrera**
**Description**

This PR removes the sidebar-writer pattern because it's virtually the same as this one but with french text. I changed a few blocks:

- The image is now the avatar
- The post meta uses the block pattern
- The useful links use the nav block instead of another query loop

**Screenshots**

Before:

Screenshot 2023-09-15 at 17 13 51

After:

Screenshot 2023-09-15 at 17 13 09

**Testing Instructions**

Create a post and select single with sidebar as the template. Make sure to add a description to the author page in your wp install to see all the content

### CODE REVIEW

File: patterns/hidden-sidebar.php [hunk 1]
```diff
@@ -8,42 +8,32 @@
...
 
 <!-- wp:separator {"backgroundColor":"base-3"} -->
 <hr class="wp-block-separator has-text-color has-base-3-color has-alpha-channel-opacity has-custom-borders-background-color has-background"/>
 <!-- /wp:separator -->
 
-<!-- wp:heading {"style":{"typography":{"fontSize":"1.6rem"}}} -->
-<h2 class="wp-block-heading" style="font-size:1.6rem"><?php echo esc_html__( 'Latest Posts', 'twentytwentyfour' ); ?></h2>
-<!-- /wp:heading -->
+<!-- wp:paragraph {"style":{"typography":{"fontSize":"1.6rem"}},"fontFamily":"cardo"} -->
+<p class="has-cardo-font-family" style="font-size:1.6rem"><?php echo esc_html__( 'Latest Posts', 'twentytwentyfour' ); ?></p>
```
Comment (MaggieCabrera) [inline patterns/hidden-sidebar.php:26]: this is coming from the editor, I don't know why it reverted to the old slugs? checking

**Comment by carolinan** [comment]
"About the author" and "Latest posts" need to remain headings.

**Comment by MaggieCabrera** [comment]
For some reason not all my changes were pushed and the markup was broken, I added the missing changes, brought back the headings and fixed the separators as per #390, please review

File: patterns/page-06-writer-home.php [hunk 2]
```diff
@@ -67,7 +67,7 @@
 <!-- wp:column {"width":"30%"} -->
 <div class="wp-block-column" style="flex-basis:30%">
 
-<!-- wp:pattern {"slug":"twentytwentyfour/sidebar-writer"}	/-->
+<!-- wp:pattern {"slug":"twentytwentyfour/sidebar"}	/-->
```
Comment (scruffian) [inline patterns/page-06-writer-home.php]:
```suggestion
<!-- wp:pattern {"slug":"twentytwentyfour/sidebar"} /-->
```

**Review by scruffian — APPROVED** [review]

**Pull request merged**